src/main/java/Duke.java

```diff
@@ -1,10 +1,177 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int DONE_START = 5;
+    public static final int TODO_START = 5;
+    public static final int EVENT_START = 6;
+    public static final int DEADLINE_START = 9;
+
+    public static Task[] tasks = new Task[100];
+    public static int numTasks = 0;
+
+    public static Scanner in = new Scanner(System.in);
+    public static String input;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        welcomeMessage();
+
+        input = getString(in);
+
+        while (!isBye()) {
+            if (isList()) {
+                executeCommand(input, Command.LIST);
+            } else if (isDone()) {
+                executeCommand(input, Command.DONE);
+            } else if (isTodo()) {
+                executeCommand(input, Command.TODO);
+            } else if (isEvent()) {
+                executeCommand(input, Command.EVENT);
+            } else if (isDeadline()) {
+                executeCommand(input, Command.DEADLINE);
+            } else {
+                executeCommand(input, Command.TASK);
+            }
+
+            System.out.println();
+            input = getString(in);
+        }
+
+
+        exitMessage();
+
     }
+
+    public static void executeCommand(String input, Command c) {
```

Comment:
It is good to refactor these functionalities

Reply:
Thank you for the comment.

Reply:
Thank you for the kind comment.